Cargo.toml

```diff
@@ -43,140 +43,140 @@ feat_acl = ["cp/feat_acl"]
 # * The selinux(-sys) crate requires `libselinux` headers and shared library to be accessible in the C toolchain at compile time.
 # * Running a uutils compiled with `feat_selinux` requires an SELinux enabled Kernel at run time.
 feat_selinux = [
-  "cp/selinux",
-  "id/selinux",
-  "ls/selinux",
-  "selinux",
-  "feat_require_selinux",
+    "cp/selinux",
```

Comment:
these changes are unrelated, please move them into a different PR

Reply:
Maybe this is just formatting issue coz I used cargo add command. I reverted the commit and added crate manually. 

I am new to this and don't have much idea.

Reply:
sorry but it still appears :)

Reply:
I will try to solve it soon.